src/main/java/com/google/devtools/build/lib/remote/GrpcRemoteCache.java

```diff
@@ -154,7 +154,10 @@ public void close() {
   }
 
   public static boolean isRemoteCacheOptions(RemoteOptions options) {
-    return options.remoteCache != null;
+    // check against 'http' rather than 'grpc' because gRPC is a default protocol when none provided
+    return options.remoteCache != null
+        && !options.remoteCache.isEmpty()
+        && !options.remoteCache.startsWith("http");
```

Comment:
Convert to lower case before comparing, as the scheme is case insensitive: https://tools.ietf.org/html/rfc3986#section-6.2.2.1

Reply:
thanks! done